src/components/Bio.tsx

```diff
@@ -15,7 +15,7 @@ const Avatar = styled.img<SizeExpecting>`
     width: ${(p) => p.size}rem;
     height: auto;
 
-    @media (max-width: ${(p) => (p.size + Constants.contentWidthPadding * 2)}rem) {
+    @media (max-width: ${(p) => (p.size + Constants.contentWidthPadding * 2)}em) {
```

Comment:
I know that I should probably know this, but I don’t usually use ems and rems very often (though I know I probably should). What led you to change the unit here?

Reply:
See my description. There's a link to an article I found.